source/extensions/filters/http/grpc_http1_reverse_bridge/filter.cc

```diff
@@ -55,7 +57,20 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::HeaderMap& headers, bool e
     return Http::FilterHeadersStatus::Continue;
   }
 
-  // TODO(snowp): Add an enabled flag so that this filter can be enabled on a per route basis.
+  // Allow enable the filter on a per route basis
+  if (decoder_callbacks_->route() && decoder_callbacks_->route()->routeEntry()) {
+    const std::string& name =
+        Extensions::HttpFilters::HttpFilterNames::get().GrpcHttp1ReverseBridge;
+    const auto* route_entry = decoder_callbacks_->route()->routeEntry();
+    const auto* config_per_route = route_entry->perFilterConfigTyped<
```

Comment:
Or I can do it this way... https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/http/ext_authz/ext_authz.cc#L48

Reply:
using that helper seems like the right thing to do since it resolves the per filter config at multiple levels

Reply:
Hi @snowp I used the helper 

```
Http::Utility::resolveMostSpecificPerFilterConfig<FilterConfigPerRoute>(
            Extensions::HttpFilters::HttpFilterNames::get().GrpcHttp1ReverseBridge,
            decoder_callbacks_->route());
```

and fixed some issues (in config files) and it seems works correctly.

Is it possible to reopen the PR or I make a new one?